inc/config.php

```diff
@@ -676,11 +676,15 @@
 	// "/```([a-z0-9-]{0,20})\n(.*?)\n?```\n?/s"
 	$config['markup_code'] = false;
 
-	// Repair markup with HTML Tidy. This may be slower, but it solves nesting mistakes. Tinyboad, at the
+	// Repair markup with HTML Tidy. This may be slower, but it solves nesting mistakes. Tinyboard, at the
 	// time of writing this, can not prevent out-of-order markup tags (eg. "**''test**'') without help from
 	// HTML Tidy.
 	$config['markup_repair_tidy'] = false;
 
+	// Use 'bare' config option of tidy::repairString.
+	// It strips out smart quotes and em dashes, etc.
+	$config['markup_repair_tidy_bare'] = false;
```

Comment:
Making it configurable is good, changing the default is not. This should be `true`

Reply:
It should not because it strips correct chars from the text. Name me a "workflow" where that behavior is appropriate.

Reply:
Wait, you are saying that this option strips out &mdash;?

Reply:
So e.g.

> &ldquo;Hello&mdash;it&rsquo;s me you&rsquo;re looking for,&rdquo;

Becomes what? Certainly not:

>Helloits me youre looking for,

right?

Reply:
Oh, did not make that clear, sorry. It makes "-" from "—".

Reply:
And this looks like complete shit.

Reply:
Vichan is legacy software. Defaults should not be changed for aesthetic reasons. As it doesn't *strip* out em dashes, but rather ASCII-izes them (and other special chars), I cannot accept this patch unless you make the default the same as it is now. Just force push a version with the default set to true. Perhaps recommend changing it in a comment.

Reply:
Well, the comment is copied straight from the Tidy CLI docs, but okay.